src/compose/ComposeText.js

```diff
@@ -85,6 +84,8 @@ class ComposeText extends React.Component {
   handleChangeText = (text: string) => {
     const { auth } = this.props;
     registerUserInputActivity(auth);
+    text = replaceEmoticonToEmoji(text);
```

Comment:
It is preferred to not mutate/reuse variables like this, but to create a new one with a good descriptive name.
It will be something like `replacedText` or maybe `emojiText`.